ui/page/root/home_page.go

```diff
@@ -523,90 +523,90 @@ func (hp *HomePage) totalBalanceLayout(gtx C) D {
 			layout.Rigid(hp.balanceLayout),
 			layout.Rigid(func(gtx C) D {
 				if hp.Load.IsMobileView() {
```

Comment:
I think the diff will be easier to follow if you reverse the if statement and return early.
```suggestion
				if !hp.Load.IsMobileView() {
```